lib/tui/update.ml

```diff
@@ -6,7 +6,9 @@ let move_fs move_fn (code_tab : Model.code_tab) =
 
 let move_issues move (issues_tab : Model.Issue.t) =
   let len = issues_tab.issues |> Lazy.force |> List.length in
-  let offset = (issues_tab.offset + move + len) mod len in
+  let offset =
+    (issues_tab.offset + move + len) mod if len = 0 then 1 else len
+  in
```

Comment:
Heh, yeah, this was a bug in the previous implementation because technically the list of issues can be zero (even without an error). I wonder how difficult it was to debug it?

But let's also rewrite it to avoid operations when `len` is zero:

```suggestion
  let offset = if len = 0 then 0 else (issues_tab.offset + move + len) mod len in
```